server/mongodb/scheduler.go

```diff
@@ -0,0 +1,88 @@
+package mongodb
+
+import (
+	"fmt"
+	"github.com/ohsu-comp-bio/funnel/compute/scheduler"
+	pbs "github.com/ohsu-comp-bio/funnel/proto/scheduler"
+	"github.com/ohsu-comp-bio/funnel/proto/tes"
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	mgo "gopkg.in/mgo.v2"
+	"gopkg.in/mgo.v2/bson"
+)
+
+// QueueTask adds a task to the scheduler queue.
+func (db *MongoDB) QueueTask(task *tes.Task) error {
+	return nil
+}
+
+// ReadQueue returns a slice of queued Tasks. Up to "n" tasks are returned.
+func (db *MongoDB) ReadQueue(n int) []*tes.Task {
+	var tasks []*tes.Task
+	err := db.tasks.Find(bson.M{"state": tes.State_QUEUED}).Select(basicView).All(&tasks)
```

Comment:
Can you select only `n` tasks? Also, probably want to order them by ID`

Reply:
Yeah I forgot about limiting to `n`.  They are ordered by ID by default.